docs/root/configuration/http/http_filters/grpc_json_transcoder_filter.rst

```diff
@@ -87,6 +87,15 @@ can be send by the gRPC server in the server streaming case.
 In this case, HTTP response header `Content-Type` will use the `content-type` from the first
 `google.api.HttpBody <https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto>`.
 
+Metadata
```

Comment:
let's call this header. `Metadata` is confusing with [dynamic metadata](https://www.envoyproxy.io/docs/envoy/latest/configuration/advanced/well_known_dynamic_metadata)

Reply:
Done. Can you review again?